iothub/device/src/InternalClient.cs

```diff
@@ -1543,20 +1543,7 @@ public Task UploadToBlobAsync(string blobName, Stream source, CancellationToken
                     throw Fx.Exception.Argument(nameof(blobName), "Path segment count cannot exceed 254");
                 }
 
-                var context = new PipelineContext();
```

Comment:
we don't need to create a new transport handler as the internal client is initialized with one in the first place.

Reply:
was this entire block added recently?

Reply:
Could you add a bit of context on what the code was previously, and what was changed?

Reply:
Or maybe just link the commits :)

Reply:
spoke to Abhipsa offline.